src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -144,4 +144,9 @@ internal partial class BoundStatementList
         protected override ImmutableArray<BoundNode> Children => 
             (this.Kind == BoundKind.StatementList || this.Kind == BoundKind.Scope) ? StaticCast<BoundNode>.From(this.Statements) : ImmutableArray<BoundNode>.Empty;
     }
+
+    internal partial class BoundPassByCopy
```

Comment:
Is there some test(s) for GetOperations?

Reply:
The node only appears in the lowered tree, so I am not sure it is observable. I have added a case here mostly out of consistency/completeness.

---
In reply to: [170112647](https://github.com/dotnet/roslyn/pull/24957#discussion_r170112647) [](ancestors = 170112647)